src/components/note/note.style.ts

```diff
@@ -37,24 +33,24 @@ const StyledNoteContent = styled.div<{
       margin-top: var(--spacing200);
     `}
   `}
+`;
 
-  ${({ hasInlineControl }) => hasInlineControl && "margin-right: 24px;"}
+const StyledNoteMain = styled.div`
+  display: flex;
+  justify-content: space-between;
+  padding-bottom: 24px;
 `;
 
 const StyledInlineControl = styled.div`
-  position: absolute;
-  top: 24px;
-  right: 16px;
-  z-index: 100;
+  display: inline-block;
+  min-width: fit-content;
 `;
 
 const StyledTitle = styled.header<{ hasInlineControl: boolean }>`
```

Comment:
```suggestion
const StyledTitle = styled.header`
```

Reply:
We can also remove `hasInlineControl` from the `StyledNoteContent` interface on line 7